turtl_backup/gui.py

```diff
@@ -52,11 +52,8 @@ def save(self):
         username = self.entry_user.get()
         password = self.entry_password.get()
         auth = turtl_backup.get_auth(username, password)
-        basic_auth = turtl_backup.build_basic_auth(auth)
         try:
-            response = requests.get(urljoin(server, '/sync/full'),
-                                    headers={'authorization': basic_auth})
-            response.raise_for_status()
```

Comment:
By removing this line, the GUI will never show the error (if an error occur while downlowding the file) and will prompt the save_as, probably crashing right after at the ` response.text` line.

You should not modify tons of things in a single PR, you started by just removing cxFreeze from the setup.py, and ended refactoring everything. I would have merged sooner if the changes were simplier, and you're still able to open multiple distincts PR for multiple unrelated modifications.

I will still merge this huge one, and thanks for all the work, once it looks OK (hint: stop modifying things on this PR, just fix my reviews).

Reply:
restored with 900176c8718a7a033a594d75b6168dcb80e2cbc3